test/unit/presenters/formats/edition_format_presenter_test.rb

```diff
@@ -74,7 +74,11 @@ def result
     should "include phase message if it's not live" do
       edition.expects(:live?).returns(false)
       edition.expects(:phase_message).twice.returns("phase message")
-      assert_equal "phase message", result[:phase_message]
+
+      phase_message = result[:details][:phase_message]
+
+      assert_equal "phase message", phase_message[:content]
```

Comment:
Minor: this test could go in the previous commit, since this commit is a refactor.